### ISSUE TITLE
CMD+OPT+ESC doesn't work while Karabiner Elements is running

### ISSUE THREAD

**Issue by oskargargas** [tekezo/Karabiner-Elements#876]
Hello,

I have 2017, touchbar, 15" MacBook Pro. System is Sierra. In Karabiner I have only two mappings: 1. right_command -> right_option and 2. right_option -> right_command. With them enabled Force Quit window does not appear after hitting CMD+OPT+ESC system shortcut. Maybe touchbar ESC has something to do with it.

Help appreciated!
Oskar

**Comment by petter-** [comment]
On my old Mac Mini 2012 with 10.12.6 the Force Quit window appears as it should.
I'm running KE 0.91.10 and have four commands

left_command -> left_option
right_command -> right_option
left_option -> left_command
right_option -> right_command

**Comment by oskargargas** [comment]
I also didn't habe such problem when I was running MacBook Air. I noticed this only this month after switching to a new MacBook Pro with a TouchBar. I'd assume there is some conflict because now ESC is virtual key on a touchbar, not physical one.

**Comment by petter-** [comment]
Ahh yes of course. I forget about the touch bar on the newer ones.

**Comment by oskargargas** [comment]
Ok, so I investigated it a bit more when I recognized that virtual (TouchBar) F-keys are working something strange/not working in IntelliJ Idea based IDEs when Karabiner is running.

In Karabiner-Elements Devices tab there are two things.
1. Apple Internal Keyboard / Trackpad v_id 1452 (which should be recognized as Apple), p_id 632. This is ticked by default
2. No product name (unknown device) v_id 1452 (again, Apple), p_id 34304. This was turned off. 

After turning it on, and turning on option to use F-keys as standard F1, F2, etc in Function Keys tab everything seems to work.

So issue is to be closed but p_id 34304 should be recognized as TouchBar and probably checked as default in addition to Internal Keyboard.
